### PR TITLE
Add annotations in helm chart

### LINKED ISSUE #200: Expose annotation setting in helm chart
## Feature Request

**Is your feature request related to a problem? Please describe:**

We could expose annotation setting in helm chart so that we can let the users configure their own annotations settings for `controller` / `daemonset` and `webhook` and their pod template.

And this would be helpful for `Chaos-Mesh` to run with `Istio` as I mentioned [here](https://github.com/pingcap/chaos-mesh/issues/146#issuecomment-582862699)

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### What problem does this PR solve? 
Fixes #200 
### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by Yisaer** [comment]
Thanks for contributing. I think the job for `pre-install` and `post-delete` in `helm chart templates` may also need to expose the pod annotation configuration. The rest LGTM. @yeya24

**Comment by yeya24** [comment]
> Thanks for contributing. I think the job for `pre-install` and `post-delete` in `helm chart templates` may also need to expose the pod annotation configuration. The rest LGTM. @yeya24

Thanks for your review, do you mean that I need to add a new section in helm `values.yaml` to expose the annotations for the jobs? So what else fields I need to add in this section?

**Comment by Yisaer** [comment]
> > Thanks for contributing. I think the job for `pre-install` and `post-delete` in `helm chart templates` may also need to expose the pod annotation configuration. The rest LGTM. @yeya24
> 
> Thanks for your review, do you mean that I need to add a new section in helm `values.yaml` to expose the annotations for the jobs? So what else fields I need to add in this section?

Yes, we need to ensure the pod created by the job won't be injected sidecar with `Istio`. That's why we also need to annotate both two jobs. @yeya24

File: helm/chaos-mesh/values.yaml [hunk 1]
```diff
@@ -89,3 +95,6 @@ dashboard:
   volume:
     storage: 3Gi
     storageClassName: standard
+
+jobs:
+  podAnnotations: {}
```
Comment (Yisaer) [inline helm/chaos-mesh/values.yaml]: Please separate `pre-install` and `post-delete`job configuration and add end-of-line in the end.

**Comment by zhouqiang-cl** [comment]
@yeya24 have you test it, LGTM

**Pull request merged**